### ISSUE TITLE
Bruh

### ISSUE THREAD

**Issue by DARKNOSY** [Smug256/Luna-Grabber-Main#3]
U not fooling me lil bro
'Builder.exe' is Creal Grabber
Webhook was already deleted anyways

https://discord.com/api/webhooks/1139520981837291521/dZWIDjR6WxAuuiNIrNr2Z3K8wyAWwZGnKzmoi92zYkaHMaMAm2t89HqYd0-tBpywgkvs

![F36RZhmW8AA60B0](https://github.com/Smug256/Luna-Grabber-Main/assets/109553205/0e67210c-ecf4-499c-8540-2b3aefd28571)

**Comment by Mehrabopera** [comment]
so... where do I get a logger lol

**Comment by DARKNOSY** [comment]
Everything from the ReadMe is from [here](https://github.com/Smug246/Luna-Grabber)
I don't know if it works anymore tho

**Comment by Dam00ny** [comment]
thx bro you saved me

**Comment by DARKNOSY** [comment]
np

**Comment by l3qso** [comment]
WWWW

**Comment by Crpvx** [comment]
darknosy?

**Comment by Crpvx** [comment]
> np

what happens when i open it?

**Comment by DARKNOSY** [comment]
Steals info from your computer and sends it to the creator

**Comment by Crpvx** [comment]
> Steals info from your computer and sends it to the creator

my friend opened it

**Comment by DARKNOSY** [comment]
Depending on when he opened it, the creator could have his credentials. 
I recommend just changing his passwords since the webhook is already deleted the creator won't have them anymore. To delete the startup program, go to setting "Startup Apps" and look for something py, cmd or something out of the ordinary

**Comment by redakgames** [comment]
Are you sure ? because today i click on this, thinking it's the real Luna Grabber, Sorry

**Comment by DARKNOSY** [comment]
I'm sure, 1st of all when someone publishes a rar, zip or exe file on github don't run it unless you have compiled it yourself using the source code. 2nd, look at the amount of stars, the issues and the creator account

**Comment by redakgames** [comment]
But does that mean he hasn't received my personal data or not ? @DARKNOSY

**Comment by DARKNOSY** [comment]
I don't really know since I didn't fully decompile it so maybe but probably not. I reccommend checking:
`C:\Users\[USER]\AppData\Local\Discord\app-[VERSION]\modules\discord_desktop_core-1\discord_desktop_core\index.js`
and if the content isn't `module.exports = require('./core.asar');`, change it to that, then check your startup apps in settings:
`Settings>Apps>Startup` then disable everything, restart your pc and then change all your passwords like discord, saved browser ones,...

**Comment by redakgames** [comment]
i verify, but yes i deleted the application of the startup but on the file "index.js" i analysed but the webhook of the virus doesn't exist. and i uninstalled and installed discord and create a new account, and delete my olds email.

**Comment by redakgames** [comment]
yes but i checked the error of the webhook on the internet, (yeah it's that, he doesnt' exist) also the informations aren't arrived to the webhook. @DARKNOSY ^_^

**Comment by DARKNOSY** [comment]
In any case if you are ever infected, I recommend factory reseting and disabling and not using onedrive because it could give you the malware back.

**Comment by redakgames** [comment]
yeah thx for the tips, in any case, he'll never have access to his future victims' data.

**Comment by DARKNOSY** [comment]
Yeah I know but it can be annoying and scary for some people to have malware
